keepassxc-browser/content/fill.js

```diff
@@ -106,7 +106,7 @@ kpxcFill.fillFromPopup = async function(id, uuid) {
 
     // For Google password field we need to do some special handling. The password field is actually in the
     // second combination that was just detected after a username fill.
-    let combination = kpxc.combinations[0];
+    let combination = kpxc.combinations[0]; // BUG? This variable is unused since https://github.com/stefansundin/keepassxc-browser/commit/ba5a7141fba456ee409446ef32da2b19c416cee3#diff-b6dc310afe6e36d10e60a76ebd2e1d07ee0eeda631e4430aeab384fbd37cdb4cL114
```

Comment:
Want to call this out. Either there's a bug right now or this unused variable can be removed.

Reply:
Must check it out if it's a leftover or something else.

Reply:
I guess this was originally for the Auto-Submit and Google, but the current version works just fine with it. So this can be removed.

Reply:
Removed. 👍